### PR TITLE
bin/etc: refactor rocksdb config

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
We should notify users to update etc/tikv.toml as well as binary.
@siddontang @disksing PTAL

### CODE REVIEW

**Comment by siddontang** [comment]
LGTM

**Comment by zhangjinpeng1987** [comment]
@disksing PTAL

File: src/bin/tikv-server.rs [hunk 1]
```diff
@@ -297,83 +338,73 @@ fn get_rocksdb_default_cf_option(matches: &Matches, config: &toml::Value) -> Roc
...
-                                           config,
-                                           Some(40960),
-                                           |v| v.as_integer());
-    opts.set_max_open_files(max_open_files as i32);
-
     opts
 }
 
+fn get_rocksdb_default_cf_option(matches: &Matches, config: &toml::Value) -> RocksdbOptions {
+    // Default column family use bloom filter.
```
Comment (disksing) [inline src/bin/tikv-server.rs]: use -> uses.

**Comment by siddontang** [comment]
CI failed @zhangjinpeng1987

**Comment by siddontang** [comment]
Ping @zhangjinpeng1987

**Pull request merged**